CMakeLists.txt

```diff
@@ -98,10 +98,10 @@ if (WITH_TESTING)
             endif()
         endforeach()
     else ()
-        file(GLOB_RECURSE TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "*_test.py")
+        file(GLOB_RECURSE TEST_OPS RELATIVE "${CMAKE_CURRENT_SOURCE_DIR}" "*_test*.py")
```

Comment:
Please run paddle tests separately according to the discussion.

Reply:
done